package.json

```diff
@@ -1,61 +1,61 @@
 {
   "name": "systelab-translate",
-  "version": "0.0.0",
+  "version": "12.0.0",
```

Comment:
No need to change it. This is not the version of the library and should be 0.0.0 to avoid confussion.

Reply:
Restored